ychat/src/commonMain/kotlin/co/yml/ychat/YChat.kt

```diff
@@ -77,6 +78,8 @@ interface YChat {
      */
     fun chatCompletions(): ChatCompletions
 
+    fun imageGenerations(): ImageGenerations
```

Comment:
I suggest adding an integration test for the `imageGenerations()` entry point that covers the integration of all layers. There is already some example you can follow on the YChatTest.

To create the mock API response body, you can store it in the `MockStorage` class and convert the JSON to a string using a site like https://jsontostring.com/. 

Here's an example format you can use to mock the response:

`{\"created\":1678805561,\"data\":[{\"url\":\"https://testlink.com/image-test.jps\"}]}`

Reply:
Done!